Example/Auth/Tests/FIRPhoneAuthProviderTests.m

```diff
@@ -34,6 +34,7 @@
 #import "FIRAuthUIDelegate.h"
 #import "FIRAuthURLPresenter.h"
 #import "FIRAuthWebUtils.h"
+#import "FIRAuthSettings.h"
```

Comment:
Nit: Ordering, the rest of the headers are alphabetical.

Reply:
Done, thanks.